locale/it_IT/common.xml

```diff
@@ -328,7 +328,9 @@
 	<message key="announcement.postedOn">Pubblicato il {$postDate}</message>
 	<message key="common.accepted">Accettato</message>
 	<message key="common.titleSmallWords">of a the and an or nor but is if then else when at from by on off for in out over to into with</message>
-	<message key="common.warning">Attenzione</message>
+	<message key="common.warning">Warning</message>
```

Comment:
Since these keys are used in log, I think that it is quite a standard to see Warning, Error or Notice in log files). So I think that it is better to not translate these lines.